test/e2e/flask/solana/create-solana-account.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Suite } from 'mocha';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import { ACCOUNT_TYPE } from '../../page-objects/common';
+import { withSolanaAccountSnap } from './common-solana';
+
+describe('Create/Remove Solana Account', function (this: Suite) {
+  it('Creates 2 Solana accounts', async function () {
+    await withSolanaAccountSnap(
+      { title: this.test?.fullTitle() },
+      async (driver) => {
+        // check that we have one Solana account
+        const headerNavbar = new HeaderNavbar(driver);
+        await headerNavbar.check_pageIsLoaded();
+        console.log('Page is loaded');
+        await headerNavbar.check_accountLabel('Solana 1');
+        console.log('Checked account label: Solana 1');
+        await headerNavbar.openAccountMenu();
+        console.log('Opened account menu');
```

Comment:
same for all other console log in this test spec file, try to add console log in class methods so they will be reusable